src/containers/GalaxiesSelectorContainer.jsx

```diff
@@ -39,17 +41,15 @@ class GalaxiesSelectorContainer extends React.PureComponent {
     const {
       widget: { source, sources },
       options,
-      answers,
     } = this.props;
+    const { savedSources } = this.global;
     const { randomSource } = options || {};
-    const randomGalaxiesAnswer = answers[this.aId];
+    const randomGalaxiesAnswer = savedSources[this.aId];
```

Comment:
updated GalaxySelector to access the new store for saved random sources instead of storing the sources in the `answer` store